src/App.jsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 
+import Header from './components/header/Header';
+import MainPage from './components/main-page/MainPage';
+
 function App() {
   return (
-    <div />
+    <div>
+      <Header />
+      <MainPage />
+    </div>
   );
 }
 export default App;
```

Comment:
you need to use named import and react memo

Reply:
+